filemanagement/filemanagement/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.IO;
+namespace filemanagement
+{
+    class Program
+    {
+        public static void enterpatient()
+        {
+            Console.Write("\nEnter Patient's Full Name:- ");
+            string patientname = Console.ReadLine();
```

Comment:
Follow coding standard local variable name should be camelCase like patientName.